src/components/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Routes, BrowserRouter, Route } from 'react-router-dom';
+import AddRoom from './Rooms/AddRoom';
+import Rooms from './Rooms/Rooms';
+import DeleteRoom from './Rooms/DeleteRoom';
+import AddUser from './user';
+import Room from './Rooms/Room';
+import Nav from './Nav';
+
+function App() {
```

Comment:
- [ ] Kindly refactor this functional component to use ES6.   Reason:  The vast majority of the software industry has deprecated the use of `function` keyword.

That way, your app will comply with the latest standards 💪